docs/docsite/rst/playbook_guide/playbooks_handlers.rst

```diff
@@ -114,6 +114,39 @@ Each handler should have a globally unique name. If multiple handlers are define
 
 There is only one global scope for handlers (handler names and listen topics) regardless of where the handlers are defined. This also includes handlers defined in roles.
 
+Controlling the notifing of handlers by defining "changed"
+----------------------------------------------------------
```

Comment:
This heading reads a little wordy to me. I don't think it would be good to make this section too repetitive with the `Defining “changed”` section. Maybe a heading like this:

"Defining when tasks change"